### ISSUE TITLE
M44 Text Log Issue

### ISSUE THREAD

**Issue by InterroLouis** [tgstation/TerraGov-Marine-Corps#1124]
## Testmerges:

Server-Side Code Modifications: https://tgstation13.org/parsed-logs/basil/server_side_modifications.dm

Round ID: 178
BYOND Version: 512.1464
Server revision compiled on: 
Master commit: 66355f5549f3d7fe82a901aeb4038ec6eb1456eb
The following pull requests are currently test merged:
#1111: 'LV Edit & Tweaks' by Alterist at commit 5d3c02c93a
#1109: 'Slight Projectile Speed Increase' by Surrealaser at commit b927f9cb99
#1091: 'Sentry Gun Rebalance 2.0' by Surrealaser at commit ae2523f3b8
#1100: 'Click refactor.' by DominikPanic at commit 814d221015
#1033: 'Tank Rebalance' by Surrealaser at commit 3d5c168565
#1107: 'Fixes initial damages not being applied on flamer_fire instanciation' by Menshin at commit f6435a0199
#1104: 'Smartgun Tweaks' by Surrealaser at commit 24082a87a8
#1121: 'Temperature defines' by InterroLouis at commit af039f1179

## Reproduction:

Basically just fire an M44 at all, and look at the text logs-each shot shows 1 less round than the gun actually has in it, resulting in both of the last shots displaying 0 rounds remaining. 
![M44_PROOF](https://user-images.githubusercontent.com/29074600/54861781-a29ec480-4d05-11e9-8a04-a5401ecf92ca.png)

**Comment by InterroLouis** [comment]
To update the issue, this seems to just affect all guns.

**Comment by Shadow-Quill** [comment]
Cannot reproduce on latest version of #827.
Likely wasn't testmerged. (And it isn't/wasn't.)

**Comment by DominikPanic** [comment]
> 
> 
> Cannot reproduce on latest version of #827.
> Likely wasn't testmerged. (And it isn't/wasn't.)

That's because there's no way that PR could have ever caused this.

**Comment by Ghommie** [comment]
It's simply because it only counts the projectiles in the magazine, excluding the chambered one.
